src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee

```diff
@@ -0,0 +1,157 @@
+
+DashboardAppsDockCtrl = ($scope, $cookies, $uibModal, $window, MnoeOrganizations, MnoeAppInstances) ->
+  'ngInject'
+
+  $scope.appDock = {}
+  $scope.appDock.isMinimized = true
+  $scope.activeApp = null
+  $scope.launchApp = {isClosed: true}
+
+  # 'Lock' the dock when a menu is expanded.
+  # Ie: we disable all effects and animation
+  $scope.appDock.isAnimated = () ->
+    $scope.activeApp == null
+
+  # Scope initialization
+  $scope.modal ||= {}
+
+  # Mobile dock: max-height need to be an absolute value for the scrolling to work
+  pageHeight = jQuery(window).height()
+  navHeight = pageHeight * 0.7
+  jQuery('#mobile-dock').css({ maxHeight: navHeight + 'px' })
```

Comment:
Can you move this DOM manipulation code in a the link function of the directive? (cf. https://docs.angularjs.org/api/ng/service/$compile#-link-)

Also, we should not be using directly jQuery but use the angular wrapper (eg. `angular.element(el)`) (cf. https://docs.angularjs.org/api/ng/function/angular.element)